recipes/default.rb

```diff
@@ -29,7 +32,7 @@
 
 directory node.loggly.tls.cert_path do
   owner 'root'
-  group 'syslog'
+  group node.rsyslog.priv_group || 'root'
```

Comment:
in the current rsyslog cookbook, there's a default of nil.  It never gets set to anything different in the CentOS settings, just in ubuntu.  I didn't want to change the default for the rsyslog coobook in this cookbook, in case someone's relying on it being nil for non-ubuntu platforms, hence the || here.  If you like I can set a loggly-specific node attribute to deal w/ this outside of the recipe.